src/components/pages/LoginForm.tsx

```diff
@@ -1,16 +1,36 @@
 import { Button, Checkbox, Form, Input } from 'antd';
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import '../../styles/forms.css'
+import { useNavigate } from 'react-router-dom'
+import { AuthContext } from '../contexts/AuthContext';
+import { userInfo } from '../../constants/UserInfo'
```

Comment:
it's always best to separate out the node package import and developer-defined modules import